### ISSUE TITLE
Smart Buttons on Single Product causing a surge of Failed Orders

### ISSUE THREAD

**Issue by bighippo999** [woocommerce/woocommerce-paypal-payments#2871]
### Describe the Bug

We've had the Smart Buttons enabled for Single Product pages since forever without issue. But the last few days we've been seeing a lot a failed orders. There's some commonalty amongst them:
the email address {name}.{6 digits}@gmail.com
they all have a company name
they all have a phone number
they are all ordering single products not variable products.

We use cloudflare, WAF, cloudflare turnstile and a few other security measures, but these are all getting through.
It appears that the buy now button on single product pages has no/little protection except that it doesn't appear for variations until options are select. It's getting clicked by whatever bots and they're giving fake info to the PayPal popup window which does ask for company a phone number (we don't require it). Via the cart/checkout would trigger form verification on our end and turnstile should prevent it, but the paypal popup is outside of our control.

### To Reproduce

It's impossible to say how to reproduce. but we're seeing this across multiple websites.

### Expected Behavior

PayPal to do a better job of detecting these junk order attempts on their form and stop submitting them as failed orders. It would be better still if the buy now button could have some protection, i.e it's not an issue on variable products.

### Actual Behavior

Lots of orders that are failing, but it's making the queues much much harder to work, we have to delete these failed orders in order to see the real orders. but they just keep coming.

### Environment

 - WordPress Version - 6.7.1
 - WooCommerce Version - 9.4.2
 - Plugin Version - 2.9.4

### Additional Details
Out of around 100 failed orders, it's not just a handful of IP addresses, or browser types, anything that we could firewall.
So we've had to turn off the buy now button on single products and that appears to have stopped it. but it's a feature we and customers do like, so we'd like to be able to turn it back on.

**Comment by sophiegyo** [comment]
I think we may be getting a few reports like these.

References:

9116596-zen
9128512-zen
9120487-zen

**Comment by SydeKrystian** [comment]
Hello @bighippo999,

Thank you for bringing this to our attention.

We are aware of an increase in similar attacks recently, particularly involving credit cards. However, this is the first instance we've encountered involving the single product page button, and your observations make sense. We'd like to investigate this further along with the logs.

While we appreciate your feedback and input, GitHub is primarily used for development-related activities. For a more detailed and prompt resolution, we kindly request that you contact our [support team](https://woocommerce.com/document/woocommerce-paypal-payments/#get-help).

When reaching out, please include a copy of your [WooCommerce system report](https://woocommerce.com/document/understanding-the-woocommerce-system-status-report/). This will help us investigate and resolve the issue more effectively.

As this falls under support, we will close this GitHub issue for now. Please rest assured that our support team will be ready to assist you when you reach out.

Thank you for your understanding!

Kind regards,
Krystian

**Comment by nvgandhi123** [comment]
Hello @SydeKrystian,

 I am managing Recaptcha for WooCommerce (https://woocommerce.com/products/recaptcha-for-woocommerce/)

Recently I got increase tickets that reCaptcha is not protecting and get lot of SPAM orders, What I found all those are mostly using PayPal for WooCommerce Plugin.

 What I found is attacker using directly API (Don't Know how) for creating orders and testing cards. For debug purpose I have write data to file for posted that but What I found It never comes to that point and by passing "woocommerce_after_checkout_validation" hooks.

All those customers are using Classic checkout so this hook at-least work to prevent spam order.


Thanks
